src/Stepper/Stepper.js

```diff
@@ -92,15 +90,13 @@ const Stepper = React.createClass({
   getInitialState() {
     return {
       hoveredHeaderStepIndex: -1,
-      muiTheme: this.context.muiTheme || getMuiTheme(),
       itemWidth: 0,
     };
   },
 
-
   getChildContext() {
     return {
-      muiTheme: this.state.muiTheme,
+      muiTheme: this.context.muiTheme,
```

Comment:
This needs to be removed -- it's already in context and it's been removed from `childContextTypes`. The component isn't currently working as `muiTheme` is not in `childContextTypes`. 

(it does not need to be either btw, so you were right in removing it from `childContextTypes` :+1: )